Swift/Tests/ReplicatorTest+CustomConflict.swift

```diff
@@ -185,6 +185,82 @@ class ReplicatorTest_CustomConflict: ReplicatorTest {
         XCTAssertNil(db.document(withID: "doc"))
     }
     
+    // TODO:: only use one helper method
```

Comment:
What does this mean?

Reply:
after merging one set of helper function, this two are common in all PRs, will remove all others.